lib/Default/WeightedRandom.class.inc

```diff
@@ -73,19 +73,15 @@ class WeightedRandom {
 		return $this->weighting;
 	}
 	
-	public function percentFailed($percent) {
-		$failed = mt_rand(0, 99) + $this->weighting >= $percent;
-		$this->adjustWeightByPercent(!$failed, $percent);
-		return $failed;
-	}
-	
-	public function adjustWeightByPercent($success, $percent) {
-		$percent = $percent / 100;
-		if ($success) {
-			$percent = -1 + $percent;
+	public function checkHit($weaponAccuracy) {
```

Comment:
This code looks equivalent to before but just inverted to me? Am I missing something as you mentioned you tweaked it

Reply:
I believe you are correct.  Looking back I suspect my testing was at fault for the numbers I was getting.